public/app/core/utils/query.ts

```diff
@@ -9,6 +9,27 @@ export const getNextRefIdChar = (queries: DataQuery[]): string => {
   }
 };
 
+export function queryIsEmpty(query: DataQuery): boolean {
+  const schema: Record<keyof DataQuery, string> = {
+    refId: 'string',
+    hide: 'boolean',
+    key: 'string',
+    queryType: 'string',
+    datasource: 'object',
+  };
+
+  for (const key in query) {
+    if (key === 'label') {
```

Comment:
Also seems like the query runner adds this `label` prop to the query. I have no idea where this is coming from.